content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -222,50 +164,123 @@ The drawbacks of ORMs are not always apparent when you start using them. This se
-- ORMs tend to have a large API surface due to the complexity they encapsulate. The flip side of not having to write SQL is that you spend a lot of time learning how to use the ORM. This applies to most abstractions, however without understanding the way the database works, understanding and improving slow queries can be difficult.
+- ORMs tend to have a large API surface due to the complexity they encapsulate. The flip side of not having to write SQL is that you spend a lot of time learning how to use the ORM. This applies to most abstractions, however without understanding how the database works, improving slow queries can be difficult.
 - Some _complex queries_ aren't supported by ORMs due to the flexibility that SQL offers. This problem is alleviated by raw SQL querying functionality in which you pass the ORM a SQL statement string and the query is run for you.
 
 Now that the costs and benefits of ORMs have been covered, you can better understand what Prisma is and how it fits in.
 
 ## Prisma
 
-Prisma is a **database toolkit** that makes working with databases easy for application developers. It currently consists of two main tools:
+Prisma is a database toolkit that makes working with databases easy for application developers. It consists of three tools:
+
+- [**Prisma Client**](../../reference/tools-and-interfaces/prisma-client/api): Auto-generated and type-safe _Data Mapper ORM_.
+- [**Prisma Migrate**](https://www.notion.so/prismaio/Public-Documentation-for-Prisma-Migrate-Early-Access-7708df0c1d3d4852ab39fb69ed182786) [(early access)](https://www.prisma.io/docs/about/releases#early-access): A declarative data modeling and migration tool.
+- [**Prisma Studio**](../../reference/tools-and-interfaces/prisma-migrate): A modern GUI for browsing and managing data in your database.
+
+The three tools use the [Prisma schema](../../reference/tools-and-interfaces/prisma-schema/prisma-schema-file) as a single source truth for the database schema, your application's object schema, and the mapping between the two. It's defined by you and is your main configuration file for Prisma.
+
+Prisma makes you productive and confident in the software you're building with features such as _type safety_ and a streamlined developer experience.
+
+In the next section, you will learn about Prisma implement the Data Mapper ORM pattern.
+
+### How Prisma implements the Data Mapper pattern
+
+The Data Mapper pattern allows the object schema to evolve independently of the database schema by separating concerns into entity and mapper classes. One of the reasons that traditional data mapper ORMs do this is due to the structure of organizations where the two responsibilities would be handled by separate teams, e.g., [DBAs](https://en.wikipedia.org/wiki/Database_administrator) and backend developers.
```

Comment:
First sentence could link back to the definition and description of that above.

Reply:
Great idea! Done.